### DISCUSSION TITLE
startup at boot for XA

### DISCUSSION THREAD

**Discussion by jberrebi** [yarra-framework/yarra-framework.github.io#7] [Q&A] [answered]
Hi,
We were running VE11C on siemens Prisma before the summer and there it was ok that yarra was not loaded at boot since medUser could open the windows start menu and start RDS without admin rights.
Now, we have upgraded to XA60 and I have tried to copy shortcuts to start menu (both to meduser's start menu and to C:/ProgramData's start menu) and nothing is loaded at boot. There is no other way than to login as medadmin. Do you know anything that could help?
Thank you,
Jonathan

**Comment by tblock79** [comment]
Unfortunately, we haven't found a way to automatically start the RDS client under XA30 or later. The only workaround is to build a "pseudo sequence" that starts the client when running the sequence, as described here:
https://yarra-framework.org/blog/running_on_xa/

**Reply by jstorrs** [reply]
Make sure you see the solution below that doesn't need the custom sequence!

**Comment by bkossows** [comment, marked as answer]
We managed to set up a new Scheduler task to start the RDS client with Syngo. First You need to add RDS.exe to trusted apps in Local Security Policy (run as admin). Then just add it to Scheduler (without admin privileges) as in the attached screenshots.

![rds](https://github.com/user-attachments/assets/70388aaf-dc97-4b60-98ef-1c060b99bd39)
![sch_1](https://github.com/user-attachments/assets/3dd0b889-5360-4e06-9268-85ee751c3982)
![sch_2](https://github.com/user-attachments/assets/4373fff9-01bc-43b7-bc8a-d12424c267a4)
![sch_3](https://github.com/user-attachments/assets/19f89d17-122d-4e73-8df6-ba1aa688f7fc)

**Reply by rwmair** [reply]
Just tried that and it worked successfully for me! Thanks so much for identifying this workaround.

**Reply by jstorrs** [reply]
This worked for me on a Skyra that recently made the VE11C to XA60A upgrade. I've only tested that logserver works. Some minor notes to fill in details I was uncertain about:

1. When setting local security policy, I was tempted to set it to the entire yarra client directory rather than only RDS.exe (%OSDRIVE%/yarra/RDS.exe vs %OSDRIVE%/yarra/\*.exe vs %OSDRIVE%/yarra/*). Currently, I only care about LogViewer on this scanner and that works fine with %OSDRIVE%/yarra/RDS.exe. Just to say if other features are goofy it might be related to that.
2. Started Local Security Policy as administrator
3. Started Task Scheduler as meduser (not administrator)

Thank you so much for this!

**Reply by tblock79** [reply]
This solution seems to work. Many thanks for sharing it!!!
